lib/bolt/cli.rb

```diff
@@ -575,10 +575,26 @@ def show_task(task_name)
       outputter.print_task_info(pal.get_task(task_name))
     end
 
+    # Filters a list of content by a filter and a list of patterns. The filter
+    # is a substring that matches the content name, whereas the patterns are
+    # glob patterns.
+    #
+    private def filter_content(content, filter, patterns)
+      if filter
+        content = content.select { |name,| name.include?(filter) }
```

Comment:
Can we just turn this into a pattern as well by wrapping it in `*`? That would have the side effect of allowing users to pass a glob filter, which doesn't seem undesirable.

Reply:
Ah I guess the problem is that `patterns` has to match _any_, but `filter` _must_ match.